### PR TITLE
Eva fix interactions

### PR DESCRIPTION
**Pull request by EvaMarco**

### CODE REVIEW

File: frontend/src/app/main/ui/viewer.cljs [hunk 1]
```diff
@@ -116,7 +117,9 @@
          (mf/deps overlay close-click-outside?)
          (fn [_]
            (when close-click-outside?
-             (st/emit! (dv/close-overlay (:id overlay))))))]
+             (if (:animation overlay)
```
Comment (hirunatan) [inline frontend/src/app/main/ui/viewer.cljs]: good place to use `(if-let`  (not very important but anyway...)

File: frontend/src/app/main/ui/viewer.cljs [hunk 2]
```diff
@@ -387,55 +389,56 @@
...
-              overlay-size
-              overlay-position))
-
-           :close-overlay
-           (let [overlay-viewport (dom/get-element (str "overlay-" (str (:overlay-id current-animation))))
-                 overlay (d/seek #(= (:id (:frame %)) (:overlay-id current-animation))
-                                 overlays)
-                 overlay-size (calculate-size (:objects page) (:frame overlay) zoom)
+       (when current-animations
+         (.log js/console (clj->js current-animations))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/viewer.cljs]: console

File: frontend/src/app/main/ui/viewer.cljs (hunk 1 above)
Comment (EvaMarco) [reply frontend/src/app/main/ui/viewer.cljs]: I need more info about this, I have change it but I don't know the differents between them :)

File: frontend/src/app/main/ui/viewer.cljs (hunk 2 above)
Comment (EvaMarco) [reply frontend/src/app/main/ui/viewer.cljs]: Ups. :)

**Pull request merged**